app/assets/javascripts/pages/differences.js

```diff
@@ -1,8 +1,8 @@
-import "/codemirror-mode.js";
+import "/codemirror-mode.js"
```

Comment:
This is a `.js` file (not `.coffee`), so those semicolons *are* valid.  And the same applies in the other `.js` files that you changed in this folder.  I assume that you know this, because the change is repeated and looks intentional.

Personally, my policy has long been to avoid JS's semicolon inference whenever possible.  To provide a mild argument as to why: In my experience, without semicolons, syntactic errors can produce some weird (read: confusing) error messages.

🤷‍♂️

Reply:
It was totally intentional.  I don't like dealing with the semicolons and haven't run into those weird, confusing errors you mention when not using them.  I'll to throw them back in.